app/controllers/events_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class EventsController < ApplicationController
+  before_action :logged_in, only: %i[new create]
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @creator = User.find(session[:user_id])
+    p @creator
```

Comment:
- [ ] Why are you printing the `@creator`? If you needed it for debugging information, please remove it.